crates/tabby/src/main.rs

```diff
@@ -64,3 +73,47 @@ macro_rules! fatal {
         })
     };
 }
+
```

Comment:
Wanna put this logging functionality into a separate file/module?

Reply:
ack